src/licensedcode/data/licenses/erlang-otp-linking-exception.LICENSE

```diff
@@ -0,0 +1,25 @@
+---
+key: erlang-otp-linking-exception
```

Comment:
You likely want to also add a few rules (can be done later) for https://github.com/massemanet/ossie/blob/c4d2dbadeb92865137aa8f039a148a758b8202ae/src/ossie_mtp3_codec.erl#L20

Reply:
https://github.com/aboutcode-org/scancode-toolkit/issues/3898

Reply:
Added thanks!